Dockerfile

```diff
@@ -1,23 +1,30 @@
-FROM python:2.7-onbuild
-
-# 1) COPY requirements.txt
-# 2) Run pip install on ^
-# 3) COPY . /usr/src/app
-
-#ENV GUNICORN_VERSION 19.3.0
+FROM python:2.7
 
 ## Install our dependencies
-
-RUN apt-get update && \
-    apt-get -y install libxml2-dev libxslt1-dev zlib1g-dev openssh-client build-essential && \
-#    pip install gunicorn==${GUNICORN_VERSION} && \
-    apt-get -y autoremove && \
-    apt-get clean
+RUN apt-get update \
+    && apt-get -y install \
+        libxml2-dev \
+        libxslt1-dev \
+        zlib1g-dev \
+        openssh-client \
+        build-essential \
+        texlive-latex-base \
+        texlive-latex-extra \
+        texlive-latex-recommended \
+        texlive-fonts-recommended \
+    && apt-get -y autoremove \
+    && apt-get clean
+
+WORKDIR /usr/src/app/readthedocs.org
 
 ## Apply our own overrides
-#COPY local_settings.py /usr/src/app/readthedocs.org/readthedocs/settings/local_settings.py
+COPY local_settings.py.example /usr/src/app/readthedocs.org/readthedocs/settings/local_settings.py
```

Comment:
I've edited it to be ours, and the gitignore makes it not track plain `local_settings.py`. I'm basically just making this repo our own.